packages/services/src/kernel/future.ts

```diff
@@ -56,6 +57,24 @@ export class KernelFutureHandler<
     return this._done.promise;
   }
 
+  /**
```

Comment:
I suppose the diff is cleaner if you just don't move these functions now. Right now, the diff has you deleting them below and adding them back in here. Probably simpler just to leave them alone in the same spot, under onIOPub?